README.md

```diff
@@ -0,0 +1,24 @@
+# ros2cli
+
+This repository contains the source code for ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
+
+## Usage
+
+Run `ros2 --help` to see all available verbs.
+
+Run `ros2 <verb> --help` for more information on individual command usage.
```

Comment:
It's possible to run either:
```
ros2 <command> --help
```
and:
```
ros2 <command> <verb> --help
```

Maybe, it would be good to explicitly add both options here.